src/libLookup/Lookup.cpp

```diff
@@ -5575,6 +5573,7 @@ void Lookup::SendTxnPacketToNodes(const uint32_t oldNumShards,
 
       P2PComm::GetInstance().SendBroadcastMessage(toSend, msg);
 
+      lock_guard<mutex> g(m_mediator.m_lookup->m_txnShardMapMutex);
```

Comment:
is this different from just using `m_txnShardMapMutex`?

Reply:
Should be the same. I can make the change.